api/src/main/java/au/org/aodn/nrmn/restapi/service/SurveyIngestionService.java

```diff
@@ -216,7 +216,10 @@ public void ingestTransaction(StagedJob job, Collection<StagedRowFormatted> vali
         var messages = Arrays.asList(rowCount + " rows of data", siteCount + " sites", surveyCount + " surveys", obsItemCount + " distinct observable items", diverCount + " divers");
         var message = messages.stream().collect(Collectors.joining("\n"));
 
-        stagedJobLogRepository.save(StagedJobLog.builder().stagedJob(job).details(message).eventType(StagedJobEventType.INGESTED).build());
+        stagedJobLog.setDetails(message);
+        stagedJobLog.setEventType(StagedJobEventType.INGESTED);
+        stagedJobLogRepository.save(stagedJobLog);
+
         job.setStatus(StatusJobType.INGESTED);
```

Comment:
this line too (although it isn't part of this PR), what if a job is marked as `INGESTED` but previous steps have failed somewhere?